sapi/phpdbg/phpdbg_out.c

```diff
@@ -872,7 +872,7 @@ PHPDBG_API int phpdbg_xml_vasprintf(char **buf, const char *format, zend_bool es
 }
 /* copy end */
 
-PHPDBG_API int _phpdbg_xml_asprintf(char **buf, const char *format, zend_bool escape_xml, ...) {
+PHPDBG_API int _phpdbg_xml_asprintf(char **buf, const char *format, unsigned int escape_xml, ...) {
```

Comment:
Not sure I understand this change - why phpdbg_xml_vasprintf which is called by this one still is zend_bool and this one is unsigned int?

Reply:
I also don't understand this part. The other change is fine, but this signature change doesn't make sense to me. It's a boolean argument, so we use zend_bool...